### DISCUSSION TITLE
Retrieve LINKS coordinates of a given gene with the peaks of a specific cluster

### DISCUSSION THREAD

**Discussion by matteozoia4** [stuart-lab/signac#1115] [Q&A] [answered]
Dear all,

I would like to retrieve LINKS coordinates of a given gene with the peaks of a specific cluster - and be able to plot them and save in a file as list both.

a) Are there steps during the peak-call (i.e group.by = celltypes/clusters) I have to perform in order to get the links to a gene in a SPECIFIC CLUSTER?

b) I am using this code but I can't add parameters such as; idents / indent.1 / position.only / to get the link with a gene in a specific cluster ?

`LinkPeaks(heart, peak.assay = "Macs2Peaks", expression.assay = "SCT", peak.slot = "counts", expression.slot = "data", gene.coords = NULL, distance = 5e+05, min.distance = NULL, min.cells = 10, genes.use = c("Shox2"), n_sample = 200, pvalue_cutoff = 0.05, score_cutoff = 0.05, gene.id = FALSE, verbose = TRUE)`

Many thanks for all the help!

MZ

**Comment by timoast** [comment, marked as answer]
Peak-to-gene linking is not done on a per-cluster basis, it uses correlation calculated across the whole dataset. If you want to find peaks specific to a certain cell type that are linked to a gene, you can find differentially accessible peaks and then intersect those with the peaks linked to genes.

**Reply by MaraLlamas** [reply]
Hi!
Thank you very much for the tools and for the great advice in these discussions.
I am having a similar issue and I wonder if it would be reasonable to separate the specific cluster and perform the peak-to-gene linking with the information of that cluster only?
Thanks again!!
